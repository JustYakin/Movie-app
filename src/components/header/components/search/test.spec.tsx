import React from "react";
import { render } from "@testing-library/react";

import Search from "./search";

describe('Test Search component', () => {
    it('Should be render component Search', () => {
        const {container} = render(<Search />);
        expect(container).toMatchSnapshot();
    });
})