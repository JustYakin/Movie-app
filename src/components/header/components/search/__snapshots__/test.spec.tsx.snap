// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Search component Should be render component Search 1`] = `
<div>
  <div
    class="search__wrapper"
  >
    <input
      class="header__input__search"
      placeholder="What do you want to watch?"
      type="text"
    />
    <button
      class="header__btn__search"
    >
      Search
    </button>
  </div>
</div>
`;
